src/modules/drawing/DrawingModule.vue

```diff
@@ -293,22 +296,44 @@ export default {
             clearTimeout(this.KMLUpdateTimeout)
             const kml = generateKmlString(this.drawingLayer.getSource().getFeatures())
             if (kml && kml.length) {
+                this.savingStatus = 'draw_file_saving'
                 try {
                     await this.saveDrawing(kml)
                 } catch (e) {
                     log.error('Could not save KML layer: ', e)
+                    // We should probably add a translation key for that
+                    this.savingStatus = 'Error while saving the drawing'
```

Comment:
Agree we MUST add a translation here.

Reply:
Right. Do you now how to do that? Can I try to add it myself or is there some kind of translation team?

Reply:
I don't know at best ask @pakb

Reply:
we could use use this existing key for now :

| key | de | en | fr | it | rm |
| ----| --- | --- | -- | -- | -- |
| upload_failed | Fehler beim Hochladen! | Upload error! | Erreur de chargement! | Caricamento fallito! | Errur da chargiar |  

Or we could add one more, but that require some translation effort